src/test/unit/ui/LoginTestPage.ts

```diff
@@ -0,0 +1,41 @@
+// src/pages/LoginTestPage.ts
+import { By, WebDriver } from 'selenium-webdriver';
+
+export class LoginTestPage {
```

Comment:
I like the separation of these elements for reuse, however I'd create a new directory for them to keep them separate from actual tests